packages/compute-baseline/README.md

```diff
@@ -40,15 +40,71 @@ Run:
 
 ## Usage
 
+### Check support for a group of compat keys
+
 ```javascript
 import { computeBaseline } from "compute-baseline";
 
 computeBaseline({
-  compatKeys: ["css.properties.border-color"],
-  checkAncestors: false,
+  compatKeys: [
+    "javascript.builtins.AsyncFunction",
+    "javascript.builtins.AsyncFunction.AsyncFunction",
+    "javascript.operators.async_function",
+    "javascript.operators.await",
+    "javascript.statements.async_function",
+  ],
 });
 ```
 
+Returns:
+
+```
+{
+  baseline: 'high',
+  baseline_low_date: '2017-04-05',
+  baseline_high_date: '2019-10-05',
+  discouraged: false,
+  support: Map(7) { … }
+  toJSON: [Function: toJSON]
+}
+```
+
+Use the `toJSON()` method to get a `web-features`-like plain JSON representation of the status.
+
+### Check support for a single support key
+
+Sometimes it can be hepful to know if parent features have less support than the specific feature you'r checking (for example, the parent is behind a prefix or flag) when computing a status for a deeply-nested feature.
```

Comment:
```suggestion
Sometimes it can be helpful to know if parent features have less support than the specific feature you're checking (for example, the parent is behind a prefix or flag) when computing a status for a deeply-nested feature.
```